keel-api/src/test/kotlin/com/netflix/spinnaker/keel/rest/ArtifactControllerTests.kt

```diff
@@ -59,9 +59,9 @@ internal class ArtifactControllerTests {
     val artifact = DeliveryArtifact("fnord", DEB)
     with(artifactRepository) {
       register(artifact)
-      store(artifact, "fnord-1.0/builds/1")
-      store(artifact, "fnord-2.0/builds/2")
-      store(artifact, "fnord-2.1/builds/3")
+      store(artifact, "fnord-1.0.0-41595c4")
+      store(artifact, "fnord-2.0.0-608bd90")
+      store(artifact, "fnord-2.1.0-18ed1dc")
```

Comment:
Slight concern that version always have to conform to a valid format in order for things to work

Reply:
This seems fine to me... 🤷‍♀

Reply:
Actually this should no longer be a problem once I addressed @cfieber's suggestion to make the comparison null-safe